test/api/encode_options_test.cpp

```diff
@@ -1033,6 +1033,13 @@ TEST_F (EncodeDecodeTestAPI, SimulcastSVC) {
       EXPECT_TRUE (iResult == cmResultSuccess) << "iResult=" << iResult << "LayerIdx=" << iIdx;
       iResult = decoder[iIdx]->DecodeFrame2 (NULL, 0, pData, &dstBufInfo_);
       EXPECT_TRUE (iResult == cmResultSuccess) << "iResult=" << iResult << "LayerIdx=" << iIdx;
+      if (dstBufInfo_.iBufferStatus == 0) {
```

Comment:
Why these changes, in the encoding options test, when you are implementing a new mode in the decoder? This feels unrelated.

Reply:
This is because there is no way to determine if there is B-frame to be present in the stream. So the ready decoded images have to be queued for reordering before they can be for output..